cookbook/assetic/jpeg_optimize.rst

```diff
@@ -250,4 +250,10 @@ file:
             ),
         ));
 
+.. tip::
+
+    Instead of using Assetic and Jpegoptim, you can also compress and manipulate
+    images before serving them using the `LiipImagineBundle`_ community bundle.
```

Comment:
Imagine will not perform the optimization for you. and this article is precisely about Assetic, so the note is misleading.

btw, compressing images dynamically instead of compressing them during the asset dumping is a bad idea for performance (the LiipImagineBundle use case is for images uploaded by users which then need to be manipulated, not really for your assets, while Assetic is about your assets)